main_test.go

```diff
@@ -144,6 +144,90 @@ func TestVersion(t *testing.T) {
 	})
 }
 
+func TestDecode(t *testing.T) {
+	Convey("For the decode subcommand", t, func() {
+		tmp := t.TempDir()
+
+		Convey("it can decode the paths in byusergroup files", func() {
+			bug := filepath.Join(tmp, "test.byusergroup")
+			writeFileStringAndCompress(t, bug, ""+
+				"user1\tgroup1\t"+encode.Base64Encode("/some/path")+"\t30\t1024\n"+
+				"user2\tgroup2\t"+encode.Base64Encode("/some/other/path")+"\t31\t1025\n"+
+				"user2\tgroup2\t"+encode.Base64Encode("/some/path/again")+"\t32\t1026\n",
+			)
```

Comment:
Should have a test for a path with \n and \t in the basename

Reply:
We do in the End2End tests, or are you asking for a 'decode' test specifically?